### DISCUSSION TITLE
In education.github my name.com and get.tech is not working

### DISCUSSION THREAD

**Discussion by Sakshamshakya319** [community/community#41844] [GitHub Education]
### Select Topic Area

Question

### Body

In my education account Name.com and get.tech is not working , they are not able to provide me domains can you help me to get my domains from these sites

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Sakshamshakya319! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
